### DISCUSSION TITLE
Carousel with remote images

### DISCUSSION THREAD

**Discussion by ChrisRisner** [nunocoracao/blowfish#730] [General]
I'm just getting started with the Hugo theme and I'd like to put a carousel on a page but reference images on a remote server by full URL.  It seems to work fine with the Gallery Shortcode but I'm not getting it to work with the Carousel.  Is there a way to make it work?

**Comment by nunocoracao** [comment]
Thanks @ChrisRisner I think this might be a bug. Would you mind opening a ticket with examples so that I can investigate?

**Reply by ChrisRisner** [reply]
Thanks!  Done!

https://github.com/nunocoracao/blowfish/issues/736
